scim/users.go

```diff
@@ -86,6 +86,12 @@ func (a UsersAPI) Patch(userID string, r patchRequest) error {
 	return a.client.Scim(a.context, http.MethodPatch, fmt.Sprintf("/preview/scim/v2/Users/%v", userID), r, nil)
 }
 
+// Disable will disable the user given the user id
+func (a UsersAPI) Disable(userID string) error {
```

Comment:
please make sure there's OpenAPI spec for this as well.

Reply:
Removed this function and just did it inline in resource_user.go. I did not mean to pollute the API space here.